/mmr_ekf_odometry_node:
  ros__parameters:
    generic:
      cones_topic: "/clusters"
      imu_topic: "/synced/imu/data"
      input_odom_topic: "/fast_limo/state"
      output_odom_topic: "/Odometry"
      gps_speed_topic: "/speed/gps"
      race_status_topic: "/planning/race_status"
      enable_logging: true
      cone_time_seen_th: 4
      is_skidpad_mission: false

    frequency:
      imu_fs: 100
      gps_fs: 10

    noises:
      #proc_noise:  [1.0, 0.5]        # Sigma range [m] and bearing [rad]
      proc_noise:  [0.8, 0.2]
      meas_noise:  [0.02, 0.02, 0.07]  # Gaussian error of the odometry X,Y position [m] and Yaw angle [rad]
      min_new_cone_distance: 2.0      # Minimum distance to create new cone